src/parser/classes/VideoDescriptionHeader.ts

```diff
@@ -0,0 +1,27 @@
+import { type ObservedArray, YTNode } from '../helpers.js';
+import Parser, { type RawNode } from '../index.js';
+import type { Thumbnail } from '../misc.js';
+import Factoid from './Factoid.js';
+
+export default class VideoDescriptionHeader extends YTNode {
+  static type = 'VideoDescriptionHeader';
+
+  channel: String;
+  channel_id: String;
+  channel_thumbnails: String[];
+  factoids: ObservedArray<Factoid>;
+  publish_date: String;
+  title: String;
+  views: String;
+
+  constructor(data: RawNode) {
+    super();
+    this.title = data.title.runs.map(({ text }: { text: String }) => text).join('');
```

Comment:
The `Text` should go here too. All instances were the text object has either a `runs` or `simpleText` should use that.